components/python/PyQt-builder/PyQt-builder.p5m

```diff
@@ -0,0 +1,87 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Daniel Chan
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+depend type=require fmri=library/python/packaging-39
+depend type=require fmri=library/python/sip
+
```

Comment:
This should be sip-39 because this package is being build with python-3.9.

Reply:
I think sip is a tool (it has a been file), so I changed it to sip, how do you think?

Reply:
That would be ok then. Did you check that sip works with other python versions, too?
We will probably drop python 3.5 as soon as IPS works with 3.9. Probably 3.7 gets dropped next and 3.10 or 3.11 added.

Reply:
Yes, tested building pyqt-builder by python3.7 and the 3.9 sip still works. I will try to make them to adapt new version of python when higher version is shipped.